### PR TITLE
Adapt install information for 3.0

### PR DESCRIPTION
**Pull request by cornelinux**
Add information about other ubuntu packages
add info about stamping

Working on #1489

### CODE REVIEW

File: doc/installation/ubuntu.rst [hunk 1]
```diff
@@ -6,10 +6,84 @@ Ubuntu Packages
...
+
+Starting with privacyIDEA 3.0 packages are available 
+for Ubuntu 16.04LTS and 18.04LTS [#ubuntu1604]_.
+
+Installing privacyIDEA 3.0 or higher
+....................................
+
+Before installing privacyIDEA 3.0 or upgrading to 3.0 you need to add the repository.
+
+.. _add_ubuntu_respository:
```
Comment (fredreichbier) [inline doc/installation/ubuntu.rst]: Typo: *repository

File: doc/installation/ubuntu.rst [hunk 2]
```diff
@@ -6,10 +6,84 @@ Ubuntu Packages
...
+If you want to upgrade your privacyIDEA on Ubuntu to privacyIDEA 3.0,
+you need to add a new repository configuration as described in 
+:ref:`_add_ubuntu_respository`.
+
+Now you can simply run:
+
+   apt update
+   apt dist-upgrade
+
+After this it is a good idea to remove old, unused packages by runnint:
```
Comment (fredreichbier) [inline doc/installation/ubuntu.rst]: *running

File: doc/installation/ubuntu.rst [hunk 3]
```diff
@@ -112,3 +186,5 @@ For further details and configuration see :ref:`otrs_plugin`.
 .. [#otrs] http://www.otrs.com/
 .. [#ubuntu1604] Starting with privacyIDEA 2.15 Ubuntu 16.04 packages are
    provided
+.. [#ubunut1804] Starting with privacyIDEA 3.0 Ubuntu 16.04 and 18.04 packages
```
Comment (fredreichbier) [inline doc/installation/ubuntu.rst]: *ubuntu

File: doc/installation/ubuntu.rst [hunk 4]
```diff
@@ -6,10 +6,84 @@ Ubuntu Packages
 
 .. index:: ubuntu
 
-There are ready made packages for Ubuntu 14.04 LTS and 16.04 LTS [#ubuntu1604]_.
-These are available in a public ppa repository [#ppa]_,
-so that the installation
-will automatically resolve all dependencies.
+There are ready made packages for Ubuntu.
+Up to privacyIDEA 2.23 packages are available for 
+Ubuntu 14.04 LTS and 16.04 LTS [#ubunut1604]_.
```
Comment (fredreichbier) [inline doc/installation/ubuntu.rst]: *ubuntu1604

File: doc/installation/ubuntu.rst [hunk 5]
```diff
@@ -6,10 +6,84 @@ Ubuntu Packages
...
+
+
+.. _upgrade_ubuntu:
+
+Upgrading privacyIDEA to 3.0
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to upgrade your privacyIDEA on Ubuntu to privacyIDEA 3.0,
+you need to add a new repository configuration as described in 
+:ref:`_add_ubuntu_respository`.
```
Comment (fredreichbier) [inline doc/installation/ubuntu.rst]: *repository

File: doc/installation/ubuntu.rst (hunk 3 above)
Comment (cornelinux) [reply doc/installation/ubuntu.rst]: done in d69da2fc

File: doc/installation/ubuntu.rst (hunk 2 above)
Comment (cornelinux) [reply doc/installation/ubuntu.rst]: done in d69da2fc

File: doc/installation/ubuntu.rst (hunk 5 above)
Comment (cornelinux) [reply doc/installation/ubuntu.rst]: done in d69da2fc

File: doc/installation/ubuntu.rst (hunk 1 above)
Comment (cornelinux) [reply doc/installation/ubuntu.rst]: done in d69da2fc

File: doc/installation/ubuntu.rst (hunk 4 above)
Comment (cornelinux) [reply doc/installation/ubuntu.rst]: done in d69da2fc

File: doc/installation/ubuntu.rst (hunk 2 above)
Comment (fredreichbier) [reply doc/installation/ubuntu.rst]: Oh whoops, seems like the commit didn't fix that typo. I pushed 7032b87 which changes this to "running".

**Comment by fredreichbier** [comment]
The changes look good to me. But as we discussed in #1127, should we also add a note to the *upgrade* documentation for pip installations that one can use ``pip install -r lib/privacyidea/requirements.txt`` to pin dependencies to the tested versions?

**Comment by cornelinux** [comment]
@fredreichbier: I actually added the pinning in the documentation!

File: doc/installation/pip.rst [hunk 6]
```diff
@@ -32,7 +32,12 @@ Now you are within the python virtual environment.
 Within the environment you can now run::
  
   pip install privacyidea
-  # TODO: Do the deterministic installation!
+
+To achieve a deterministic installation, you can now pin the installed
+versions to our tested versions like this::
+
+  pip install -r lib/privacyidea/requirements.txt
```
Comment (plettich) [inline doc/installation/pip.rst]: will this be available in the next privacyIDEA pypi release?

Comment (plettich) [reply doc/installation/pip.rst]: sorry, didn't read the issue.

**Comment by fredreichbier** [comment]
Instead of adding a hint, we change the behavior of privacyidea-pip-update. I opened #1555 for that.

**Pull request merged**